CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+<!--
+
+    Copyright (c) 2019-present Sonatype, Inc.
+    This program and the accompanying materials are made available under
+    the terms of the Eclipse Public License 2.0 which accompanies this
+    distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+
+-->
+
+# How to be a contributor to this project
+
+## Talk to a human first
+
+### For Sonatypers
+
+Looking to dive in? FANTASTIC! Reach out to one of our experts and we can help get you started in the right direction.
+
+* Hit us up on our Slack channel #integrations.
+* Please fill out an issue (Jira INT project) for your PR so that we have traceability as to what you are trying to fix,
+  versus how you fixed it.
+
+### For Non-Sonatypers
+
+* Come hang out with us at our [gitter channel](https://gitter.im/sonatype/nexus-developers) so we can try to understand the problem you are trying to solve.
+* Sign the [Sonatype CLA](https://sonatypecla.herokuapp.com/sign-cla)
+
+## Submitting a PR
+
+* Add the `@sonatype/integrations` group as a reviewer on the PR
```

Comment:
Is this still an outstanding question? I would assume that this would be an internal Slack group. @mpuglin can you verify this? Is this group visible to GitHub users outside of our org?

Reply:
> 
> 
> Is this still an outstanding question? I would assume that this would be an internal Slack group. @mpuglin can you verify this? Is this group visible to GitHub users outside of our org?

I briefly tried to cite that group when I logged in as a non-sonatype account and it was not visible. This is useful for Sonatype only. I updated it to try to clarify